demo_setup.py

```
@@ -7,144 +7,162 @@
+
         git_manager = GitSetupManager(temp_dir)
-        
+
         # Check if it's a git repo
         is_repo = git_manager.is_git_repo()
         console.print(f"🔍 Is Git repo: {'✅ Yes' if is_repo else '❌ No'}")
-        
+
         if not is_repo:
             console.print("🚀 Initializing Git repository...")
             subprocess.run(["git", "init"], cwd=temp_dir, check=True)
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)